xclim/indices/fwi.py

```diff
@@ -920,23 +926,26 @@ def fire_weather_ufunc(
     indexes : Sequence[str], optional
         Which indexes to compute. If intermediate indexes are needed, they will be added to the list and output.
     season_method : {None, "WF93", "LA08", "GFWED"}
-        How to compute the start up and shut down of the fire season.
-        If "None", no start ups or shut downs are computed, similar to the R fwi function.
+        How to compute the start-up and shut-down of the fire season.
+        If "None", no start-ups or shut-downs are computed, similar to the R fwi function.
```

Comment:
```suggestion
        How to compute the start-up and shutdown of the fire season.
        If "None", no start-ups or shutdowns are computed, similar to the R fwi function.
```
That's what Merriam says!